scripts/utils/update_package.py

```diff
@@ -164,6 +165,41 @@ def get_increased_version(url, version):
     return (None, None)
 
 
+def get_msixbundle_version(url, version):
+    if not url.endswith(".msixbundle"):
+        return (None, None)
+
+    pack_name = url.split("/")[-1].replace(".msixbundle", "")
+
+    resp = requests.get(f"https://aka.ms/{pack_name}/download")
+    if not resp.ok:
+        return (None, None)
+
+    namespace = "http://schemas.microsoft.com/appx/appinstaller/2018"
+    namespaces = {"ai": namespace}
+    app_installer_version = None
+    msixbundle_uri = None
+    try:
+        xml = ET.fromstring(resp.content.decode("utf-8"))
+        app_installer_version = xml.get("Version")
+        main_bundle_element = xml.find("ai:MainBundle", namespaces)
+        if main_bundle_element is not None:
+            msixbundle_uri = main_bundle_element.get("Uri")
+        else:
+            return (None, None)
+
+        latest_sha256 = get_sha256(msixbundle_uri)
+        if latest_sha256:
+            latest_version = app_installer_version.replace(".", "-")
+            return (latest_version, latest_sha256)
+    except ET.ParseError as e:
+        print(f"Error parsing XML: {e}")
+    except Exception as e:
+        print(f"An unexpected error occurred: {e}")
```

Comment:
Our CI expects this script to only print the version: https://github.com/mandiant/VM-Packages/blob/4dfdeafdee66d358a2a4e6ebfbace2db5cd0427d/.github/workflows/update_packages.yml#L31 I think this is not documented in the script (you can add a comment to document it it if you wish). So, we must just return if an Exception occurs instead of printing an error.

[nitpick] Is there a reason to catch all exceptions in addition to `ET.ParseError`? This could hide unexpected error we may want to fix. :thinking: